File: A's/py/734A.py
def main() -> None:
    n: int = int(input())
    s: str = str(input())

    a: int = 0
    d: int = 0

    for c in s:
        if c == "A": a += 1
        else: d += 1

    if a > d: print("Anton")
    elif d > a: print("Danik")
    else: print("Friendship")

if __name__ == "__main__":
    main()

File: A's/cpp/734A.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n; cin >> n;
    string s; cin >> s;

    int a = 0;
    int d = 0;
    for (char c : s) {
        if (c == 'A') { a++; }
        if (c == 'D') { d++; }
    }

    if (a > d) { cout << "Anton" << endl; }
    else if (d > a) { cout << "Danik" << endl; }
    else { cout << "Friendship" << endl; }
    
    return 0;
}